pennylane/tape/tapes/cv_param_shift.py

```diff
@@ -402,18 +433,20 @@ def parameter_shift_var(self, idx, device, params, **options):
 
         Args:
             idx (int): trainable parameter index to differentiate with respect to
-            device (.Device): a PennyLane device that can execute quantum operations and return
-                measurement statistics
             params (list[Any]): the quantum tape operation parameters
 
         Keyword Args:
             force_order2 (bool): iff True, use the order-2 method even if not necessary
+            device (.Device): A PennyLane device that can execute quantum operations and return
+                measurement statistics. This keyword argument is required, as the device labels
+                are required to generate the quantum tapes for computing the gradient.

```

Comment:
see above; in thise case, we don't even seem to need the wires

Reply:
same as above, it seems this is required more for PolyXP implementation details, not for labels